eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -81,6 +81,9 @@ case class NodeParams(keyManager: KeyManager,
                       maxPendingPaymentRequests: Int,
                       maxPaymentFee: Double,
                       minFundingSatoshis: Long) {
+
+  require(alias.getBytes("UTF-8").length <= 32, "Invalid alias, too long")
```

Comment:
They could be a  `require` here, but the actual message should be in `makeNodeParams` for consistency.

Also "alias must be shorter than 32B"  would be more informative